CMakeLists.txt

```diff
@@ -454,6 +457,16 @@ add_compile_definitions(FOLLY_HAVE_INT128_T=1)
 set_source(folly)
 resolve_dependency(folly)
 
+# Spark qury runner depends on absl, c-ares, grpc.
+set_source(absl)
+resolve_dependency(absl 20240116 EXACT)
+
+set_source(c-ares)
+resolve_dependency(c-ares)
+
+set_source(gRPC)
+resolve_dependency(gRPC 1.48.1 EXACT)
```

Comment:
why the exact version?

Reply:
Followed the version in cmake. Removed the version requirement.
https://github.com/facebookincubator/velox/blob/bada8f2ac0d901de72686fdff984041c460c94fa/CMake/resolve_dependency_modules/google_cloud_cpp_storage.cmake#L17